control/setup_control.go

```diff
@@ -28,8 +28,9 @@ const (
 var (
 	loopFrequency   = 50.0
 	sumIndex        = 1
-	linearPIDIndex  = 2
-	angularPIDIndex = -1
+	LinearPIDIndex  = 2
+	AngularPIDIndex = -1
```

Comment:
since these are getting exported now should they be constants? not the biggest fan of using them in sensorcontrolled base, but I also get we don't have a way to know which signal is which

Reply:
yeah this is one of the things I want to change/figure out a different way to handle. there should be a better way to use these values and not have them be exported. first idea off the top of my head is to just make a helper function that is like `getBlockIndex` or something that loops through the blocks and gets whatever index you request, but that's a lot of (maybe unnecessary?) looping so idk

Reply:
so it might be better to keep it simple for now, and once we go to use MIMO with the base we can think about this more then

Reply:
okay I'm fine with that. I do think a lot will change once we add MIMO so that sounds good to me